src/libraries/Microsoft.PowerFx.Core/IR/Nodes/BinaryOpKind.cs

```diff
@@ -23,7 +23,9 @@ internal enum BinaryOpKind
         AddDecimals,
 
         DateDifference,
-        TimeDifference,
+        TimeDifference,
+        DateDifferenceDecimal,
+        TimeDifferenceDecimal,
```

Comment:
We shouldn't need this - depending on the flags the result will always be the same type

Reply:
Done